proxy/src/managers/traits/request.rs

```diff
@@ -0,0 +1,6 @@
+use sam_common::AccountId;
+
+pub trait KeyRequestManager: Send + Sync + Clone {
+    fn store_request(&mut self, sender: AccountId, receiver: AccountId);
+    fn get_requests(&mut self, account_id: AccountId) -> Option<Vec<AccountId>>;
```

Comment:
Get receivers?

Also: Option<Vec<_>> can just be vec<_> because you need to check a vec for emptiness anyway

Reply:
I return a Option<Vec<>> because there is a posibility that there is not an key value pair that is in the hashmap, then it will return None.